metrics/web/lib/common/presentation/navigation/state/navigation_notifier.dart

```diff
@@ -215,7 +215,10 @@ class NavigationNotifier extends ChangeNotifier {
 
     _currentConfiguration = newConfiguration;
 
-    final newPage = _pageFactory.create(_currentConfiguration);
+    final newPage = _pageFactory.create(
+      _currentConfiguration.name,
+      _currentPageParameters,
+    );
 
     _updatePageParameters();
```

Comment:
🔨
It seems like we should update the page parameters firstly, and only then create a page using the updated page parameters. What do you think?